src/components/description-data/index.js

```diff
@@ -0,0 +1,61 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import '../grid';
+import styles from './styles.scss';
+import './components/description-data-item';
+
+export class VlDescriptionData extends LitElement {
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      size: { type: String, attribute: 'data-vl-items-size', reflect: true },
+      maxSize: { type: String, attribute: 'data-vl-items-max-size', reflect: true },
+      mediumSize: { type: String, attribute: 'data-vl-items-medium-size', reflect: true },
+      mediumMaxSize: { type: String, attribute: 'data-vl-items-medium-max-size', reflect: true },
+      smallSize: { type: String, attribute: 'data-vl-items-small-size', reflect: true },
+      smallMaxSize: { type: String, attribute: 'data-vl-items-small-max-size', reflect: true },
+      extraSmallSize: { type: String, attribute: 'data-vl-items-extra-small-size', reflect: true },
+      extraSmallMaxSize: { type: String, attribute: 'data-vl-items-extra-small-max-size', reflect: true },
+    };
+  }
+
+  firstUpdated() {
+    const observer = new MutationObserver(() => {
+      this.requestUpdate();
+    });
+
+    observer.observe(this, { subtree: true, childList: true });
+  }
+
+  render() {
+    return html`<div class="vl-description-data">
+      <div is="vl-grid">
+        ${[...this.children].map((child, index) => {
+          const name = `item-${index}`;
+          child.setAttribute('slot', name);
+          return html`<div
+            is="vl-column"
+            data-vl-size=${this.size ? this.size : 12 / this.children.length}
```

Comment:
Stel iemand gebruikt gewoon de breadcrumb zonder al deze attributen, dan gaat het op deze manier niet werken. Als je in de story eens alle attributen af de component haalt krijg je dit in de
browser (foto). Een oplossing ervoor is dat je ze enkel zet als die prop defined is -> https://lit.dev/docs/templates/directives/#ifdefined
![Schermafbeelding 2022-02-18 om 14 14 35](https://user-images.githubusercontent.com/22538537/154689370-6e55df36-0f38-4fd0-b349-240e17507ac2.png)